packages/app/components/creator-channels/messages.tsx

```diff
@@ -536,7 +545,7 @@ export const Messages = memo(() => {
           editMessage={editMessage}
           isUserAdmin={isUserAdmin}
           keyboard={keyboard}
-          latestPaidNFTSlug={latest_paid_nft_slug}
+          edition={edition}
```

Comment:
why did you remove latestPaidNFTSlug? This is required?

Reply:
Ok I have seen the changes, but is that safe enough?

Reply:
But I am using the channel by detail endpoint to determine if there is any paid nft, why should there be the need to use the edition for it?

Reply:
We need the edition to trigger the pay flow directly instead of going to the drop page on `Collect to unlock` button tap. The edition is required by our Pay button component. If the channel by detail endpoint can also return star drop edition that would save us 2 requests that I am [making.](https://github.com/showtime-xyz/showtime-frontend/pull/2345/files#diff-35e8b62f9b70f65e462c40a883cc47ea668799d3de4585b0d46b0c098b8b6b8fR189)